assets/week4/presentation-grading.md

```diff
@@ -0,0 +1,41 @@
+# Project presentation evaluation
+
+You will have to present your project to the class during week 4. This
+presentation will be graded and will count for 28% of your final grade.
+
+Grading criteria will be evaluated using a 3-level scale:
+
+* 1: does not meet expectations
+* 2: partially meets expectations
+* 3: totally meets expectations
+
+Grades will be given by several instructors and averaged to obtain the
+final presentation grade.
+
+## Use of best practices (1-3)
```

Comment:
It is unclear to me what we will label as "best practices" in week 1: 
How about : "the project is using adequate tools and follow some open science best practices"  ?

Reply:
sure, is it better now?

Reply:
yes - thanks !